### ISSUE TITLE
#1342482654------------- Elements must have sufficient color contrast

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#528]
<strong>Source:</strong>`<span class="c-coordinateSummaryPanel__summaryStatsNum c-coordinateSummaryPanel__summaryStatsNum--blue t-jobSummaryTotal2">11</span>`
<strong>Element Location: </strong>.t-jobSummaryTotal2
<strong>Impact: </strong>serious
<strong>failureSummary:</strong> Fix any of the following:
  Element has insufficient color contrast of 4.14 (foreground color: #007ac2, background color: #f3f3f3, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
Tags: cat.color,wcag2aa,wcag143,ACT
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
